pkgs/applications/editors/jetbrains/readme.md

```diff
@@ -3,26 +3,37 @@ The jdk is in `pkgs/development/compilers/jetbrains-jdk`.
 
 ## Tests:
 - To test the build process of every IDE (as well as the process for adding plugins), build `jetbrains.plugins.tests.empty`.
-- To test the build process with all plugins\* supported by all IDEs, build `jetbrains.plugins.tests.all`.
-- To test only plugins for a specific IDE\*, build `jetbrains.ide-name.tests.plugins.all`.
 - To test that plugins are correctly stored in the plugins directory, build `jetbrains.plugins.tests.stored-correctly`.
 
-\*: Plugins marked as broken in nixpkgs are skipped: When updating/fixing plugins, please check the `broken-plugins` in `plugins/tests.nix` and update it if needed.
-
 ## How to use plugins:
- - Get the ide you want and call `jetbrains.plugins.addPlugins` with a list of plugins you want to add.
- - The list of plugins can be a list of ids or names (as in `plugins/plugins.json`)
- - Example: `jetbrains.plugins.addPlugins jetbrains.pycharm-professional [ "nixidea" ]`
- - The list can also contain drvs giving the directory contents of the plugin (this is how you use a plugin not added to nixpkgs) or a single `.jar` (executable). For an example, look at the implementation of `fetchPluginSrc` in `plugins/default.nix`.
+ - Get the IDE you want and call `jetbrains.plugins.addPlugins` with a list of plugins you want to add.
+ - The list has to contain contain drvs giving the directory contents of the plugin or a single `.jar` (executable).
+
+Nixpkgs no longer bundles plugins, however you can use third-party plugin sources, such as
```

Comment:
1. No need for "no longer"; we should document what is not what was
2. Slight wording suggestions

```suggestion
Nixpkgs does not package Jetbrains plugins, however you can use third-party sources, such as
```

Also, we should try to follow [one sentence per line](https://github.com/NixOS/nixpkgs/blob/master/doc/README.md#one-sentence-per-line) in markdown docs, such as this file. At least in new additions.